pkgs/servers/pulseaudio/default.nix

```diff
@@ -1,9 +1,13 @@
-{ lib, stdenv, fetchurl, pkg-config, autoreconfHook
+{ lib, stdenv, fetchurl, fetchpatch, pkg-config
 , libsndfile, libtool, makeWrapper, perlPackages
 , xorg, libcap, alsa-lib, glib, dconf
 , avahi, libjack2, libasyncns, lirc, dbus
 , sbc, bluez5, udev, openssl, fftwFloat
 , soxr, speexdsp, systemd, webrtc-audio-processing
+, check
+, meson
+, ninja
+, m4
```

Comment:
All these new inputs are semantically related build tools, I suggest grouping them on a single line.
Alternative: Run `nixpgs-fmt` on it as a 2nd separate commit once everything works, this enforces a *one input per line* style.